stable/etcd-operator/templates/restore-operator-service.yaml

```diff
@@ -15,6 +15,6 @@ spec:
     name: http-etcd-restore-port
     port: {{ .Values.restoreOperator.port }}
   selector:
-    app: {{ template "etcd-restore-operator.name" . }}
+    app: {{ template "etcd-restore-operator.fullname" . }}
```

Comment:
Should be `name`.

Reply:
the purpose of the change was to match the pod label for `etcd-restore-operator` [here](https://github.com/kubernetes/charts/blob/master/stable/etcd-operator/templates/restore-operator-deployment.yaml#L18) . I was seeing an issue where no endpoints were added for restore operator service, because they selector didn't match the actual pod labels. Are you saying we should change restore operator `app` label to `name` instead?